Feature/106/v2.2.0 docs (#107)
sakuli/sakuli.github.io: content/docs/01_getting_started.md

> ## Setup your first test

Reviewer: ```suggestion
## Setup Your First Test
```

Since v2.2.0 Sakuli provides an easy to use mechanism to initialize testcases and testsuites.

{{< highlight bash >}}
npx sakuli create project . my-sut
{{< /highlight >}}

> This will create all necessary files and folders to start writing your first Sakuli test right away. With this you can skip the next section and directly start to [write your frist test](#write-your-first-test)

Reviewer: ```suggestion
This will create all necessary files and folders to start writing your first Sakuli test right away. With this, you can skip the next section and directly start to [write your first test](#write-your-first-test)
```

> ### Setup the Project manually

Reviewer: ```suggestion
### Setup the Project Manually
```

With this in mind, we can add a testcase file:

{{< highlight bash >}}
mkdir case1
echo > case1/check.js
{{< /highlight >}}

And add the following information to the `testsuite.suite` file:

{{< highlight bash >}}
echo case1/check.js https://sakuli.io > testsuite.suite
{{< /highlight >}}

> ## Write your first Test

Reviewer: ```suggestion
## Write Your First Test
```

> After the setup you can add the actual testcode to `case1/check.js`:

Reviewer: ```suggestion
After the setup you can add the test code to `case1/check.js`:
```

2. To provide Sakuli information about our actual testcase, we create a TestCase object, which handles the execution of a testcase.
3. If any error occurs during testing, it will be redirected to the TestCase object. It triggers Sakuli's internal error handling e.g. taking a screenshot of the actual failed test execution.
4. Regardless of a failed or passed test execution, Sakuli saves all results. This is more like a legacy artifact and will be removed in the future.

> 5. The `()` at the end of this line will invoke the defined function. Since v2.2.0 Sakuli is able to detect this invocation without explicitly chaining the call of `done` function like in previous versions (it's still possible to add `.then(done)`);

Reviewer: ```suggestion
5. The `()` at the end of this line will invoke the defined function. Since v2.2.0 Sakuli is able to detect this invocation without explicitly chaining the call of `done` function like in previous versions (it is still possible to add `.then(done)`);
```
